model/schema/expression.yaml

```diff
@@ -44,104 +44,251 @@ classes:
+      whereExpressed.anatomicalStructureQualifierTermId:                  ExpressionAnnotation.expression_pattern (ExpressionPattern.AnatomicalSite.anatomical_structure_qualifiers)
+      whereExpressed.anatomicalStructureUberonSlimTermIds: (*REQUIRED)    ExpressionAnnotation.expression_pattern (ExpressionPattern.AnatomicalSite.anatomical_structure_uberon_terms)
+      whereExpressed.anatomicalSubStructureTermId:                        ExpressionAnnotation.expression_pattern (ExpressionPattern.AnatomicalSite.anatomical_substructure)
+      whereExpressed.anatomicalSubStructureQualifierTermId:               ExpressionAnnotation.expression_pattern (ExpressionPattern.AnatomicalSite.anatomical_substructure_qualifiers)
+      whereExpressed.anatomicalSubStructureUberonSlimTermIds:             ExpressionAnnotation.expression_pattern (ExpressionPattern.AnatomicalSite.anatomical_substructure_uberon_terms)
+      whereExpressed.cellularComponentTermId: (*REQUIRED)                 ExpressionAnnotation.expression_pattern (ExpressionPattern.AnatomicalSite.cellular_component)
+      whereExpressed.cellularComponentQualifierTermId:                    ExpressionAnnotation.expression_pattern (ExpressionPattern.AnatomicalSite.cellular_component_qualifiers)
+      whereExpressed.whereExpressedStatement (REQUIRED):                  ExpressionAnnotation.where_expressed_statement
+      crossReference:                                                     ExpressionAnnotation.data_provider.cross_reference
+      * NOTE: For the FMS whereExpressed component, either of the following is required.
+      1) anatomicalStructureTermId AND anatomicalStructureUberonSlimTermIds, or ...
+      2) cellularComponentTermId.
+      In this LinkML model, the where_expressed slot is required, with further
+      postconditions requirements in the AnatomicalSite object, though these
+      postconditions are still not as specific as the FMS model.
+      New required slots: relation.
+      New non-required slots: curie, unique_id, mod_entity_id, mod_internal_id,
+      related_notes, expression_qualifiers, negated, uncertain,
+      related_figures, expression_annotation_image_associations,
+      condition_relations.
+    slot_usage:
+      expression_annotation_subject:
+        description: >-
+          The gene to which the expression annotation applies.
+        range: Gene
+      # ACTION ITEM: Create is_expressed_in VocabularyTerm (Daniela).
+      relation:
+        description: >-
+          For a gene, the relation should be constrained to the expresssed_in
+          VocabularyTerm.
+
+  # DQM NOTE: Each FMS expression annotation is limited to one stage. However,
+  #           in the LinkML model, it's possible to specify a range.
+  ExpressionPattern:
+    is_a: AuditedObject
+    description: >-
+      An expression pattern, as defined by spatial and temporal attributes,
+      including all qualifiers.
+    slots:
+      - when_expressed
+      - where_expressed
```

Comment:
I see that where_expressed is considered a required slot but when_expressed is not. Is it true that an AnatomicalSite would always be required, but a TemporalContext would not? Maybe this is meant to have a post_conditions statement indicating that either one or the other is required?